push-webapp/index.js

```diff
@@ -16,6 +16,7 @@ async function run() {
     const service = core.getInput('service');
     const indexFile = core.getInput('indexFile');
     const templated = core.getInput('templated') === 'true';
+    const compressed = core.getInput('compressed');
```

Comment:
I think you need to add this to the inputs in action.yml as well: https://github.com/BrandwatchLtd/slipstream-actions/blob/35a7d5e195447bdccba0364751f63e7f23efb55f/push-webapp/action.yml#L3

```yaml
  compressed:
    description: ''
    required: false
    default: 'true'
```


And it's a string, so you need to do some like in the line above:

```suggestion
    const compressed = core.getInput('compressed') === 'true';
```